projects/netgrif-components/src/lib/data-fields/text-field/html-textarea-field/html-textarea-field.component.html

```diff
@@ -3,7 +3,7 @@
                   [modules]="quillModules" [bounds]="quillContainer"></quill-editor>
     <mat-error *ngIf="textAreaField.isInvalid(formControlRef)">{{getErrorMessage()}}</mat-error>
     <input type="hidden" [formControl]="formControlRef">
-    <div *ngIf="formControlRef.disabled" class="border ql-snow user-select-auto">
-        <div class="ql-editor" [innerHTML]="disabledDisplay"></div>
+    <div *ngIf="formControlRef.disabled" class="ql-snow user-select-auto" [ngClass]="{'border' : textAreaField.materialAppearance === 'outline'}">
```

Comment:
comparison against a constant instead of a string

Reply:
it is hard to put enum into HTML, you need to create a variable in .ts for that , and I think its not nice to put there "dead" variables for it